app/components/SettingsScreen/SettingsScreen.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PageHeader from '../PageHeader/PageHeader';
+import PageMain from '../PageMain/PageMain';
+
+export default function SettingsScreen({ setBoardColor, }) {
+  const headerProps = { instructions: [ 'Please enter the colors of your choice in hex format' ], };
```

Comment:
UX: "Please" is rarely used nowadays in microcopy texts. It doesn't add any value.

Reply:
done